src/Keboola/GoogleDriveWriter/Writer.php

```diff
@@ -100,21 +95,25 @@ public function processFiles($filesConfig)
 
     public function processTables($tables)
     {
-        return array_map(function ($table) {
-            $action = $table['action'];
-            if (!in_array($action, [ConfigDefinition::ACTION_CREATE, ConfigDefinition::ACTION_UPDATE])) {
+        return array_map(
+            $this->exceptionHandleWrapperFunc(function ($table) {
```

Comment:
osobně bych tady byl spíš pro přepsání na obyčejný foreach. Takhle funkcionálně je to sice elegantní, ale je to špatně čitelné. 

Mohlo by to být 
```
foreach ($filteredTables as $table) {
  $result[] = $this->processTable($table); // a tam už veškerý kód vč. exception handlingu a vybírání která action se má spustit. 
}
```